### ISSUE TITLE
Feature Request/Idea:

### ISSUE THREAD

**Issue by zbenta** [IQSS/dataverse#9305]
**Overview of the Feature Request**

Have the possibility to manipulate the presentation of messages sent by dataverse to the users.

**What kind of user is the feature intended for?**

Authenticated users.

**What inspired the request?**

The lack of possibilities to customize the messages.

**What existing behavior do you want changed?**

We would like to have the possibility to customize the messages with an institutional icon, alter the specific text and also apply some css to make look and feel more personal.

**Any brand new behavior do you want to add to Dataverse?**

The possibility to customize all messages dataverse sends to users, wether it is regarding the creation of an account, the creation of a new dataset/dataverse, granting of new permissions, etc...

**Any related open or closed issues to this feature request?**

https://github.com/IQSS/dataverse/issues/9220

**Comment by pdurbin** [comment]
@zbenta two quick things:

- When you say "messages" do you mean both the email notifications and the in-app notifications (in the upper right)? Note that emails are currently sent in plain text so switching to HTML email might be a prerequisite.
- Can you please add a title for this issue?

**Comment by JoaoMFCardoso** [comment]
> @zbenta two quick things:
> 
> * When you say "messages" do you mean both the email notifications and the in-app notifications (in the upper right)? Note that emails are currently sent in plain text so switching to HTML email might be a prerequisite.

I can clarify on this.
The idea is exactly that. To be able to customize both the in-app notifications as well as system email notifications.

> * Can you please add a title for this issue?

I can't add the title. But I'd suggest 'Notification Customization'

**Comment by cmbz** [comment]
2025/03/03
- Note: this feature was originally intended not to be very customizable
- Allowing some modifications are relatively easy to support, but some are more difficult
- Requires more detailed information about the specific changes requested
  - @zbenta could you please share more details about the specific customizations you'd like to see?

**Comment by qqmyers** [comment]
FWIW: Dataverse supports changing the strings used for notifications and emails but it does not allow adding new parameters (e.g. the dataset DOI etc. included in messages). CSS customization is also supported.

**Comment by bryanmgee** [comment]
Hi y'all, just curious if there's any update on this (think it may also relate to #10595 and more vaguely to #8364). For us at UT Austin, the emails are very hard to read because of how our Outlook system is set up to check and display any links entered directly into the body, and I imagine that this is not exclusive to us given how universities handle email security. An example:

> Hello, 
> Your dataset named UNICORN: KPI dataset for URLLC Traffic in O-RAN (view at https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdataverse.tdl.org%2Fdataset.xhtml%3FpersistentId%3Ddoi%3A10.18738%2FT8%2FFJXA40&data=05%7C02%7C%7Cebd48542e692472ec60108ddf969b86d%7C31d7e2a5bdd8414e9e97bea998ebdfe1%7C0%7C0%7C638940954641446260%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=fr9hjQHsdcV9%2BANh1pC1NOiEo46FQVwUpKUe3s31sLg%3D&reserved=0 ) was published in University of Texas at Austin Dataverse Collection (view at https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdataverse.tdl.org%2Fdataverse%2Futexas&data=05%7C02%7C%7Cebd48542e692472ec60108ddf969b86d%7C31d7e2a5bdd8414e9e97bea998ebdfe1%7C0%7C0%7C638940954641468523%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=CY%2Fhht2t%2BY5k2F4grs5ffEGR25q3dfi871s2gLfEhVM%3D&reserved=0 ).

In general, we'd benefit from the ability to customize the content and formatting of certain emails in order to be able to: 1) add parameters (e.g., DOI), 2) change or add some plain text (which sounds like it's possible now but maybe not that useful if new parameters can't be added); and 3) reformat hyperlinks/hyperlinked parameters to try and avoid email rendering like the example above (since I likely cannot convince our IT department to change link screening).

**Comment by pdurbin** [comment]
I don't believe there are any updates but in the context of https://github.com/IQSS/dataverse-frontend/pull/825 we reminded ourselves that there's a lot of interest in being able to customize notifications.
